rosidl_generator_py/resource/_msg.py.template

```diff
@@ -145,30 +149,31 @@ class @(spec.base_type.type)(metaclass=Metaclass):
 @[  end if]@
         assert isinstance(value, type(None)) or \
 @[  if field.type.is_array]@
-            ((isinstance(value, Sequence) or isinstance(value, Set) \
-            or isinstance(value, UserList)) \
-            and not isinstance(value, str) and not isinstance(value, UserString) \
+            ((isinstance(value, Sequence) or
+              isinstance(value, Set) or
+              isinstance(value, UserList)) and
+             not isinstance(value, str) and
+             not isinstance(value, UserString) and
 @[    if field.type.array_size]@
 @[      if field.type.is_upper_bound]@
-            and len(value) <= @(field.type.array_size) \
+             len(value) <= @(field.type.array_size))
 @[      else]@
-            and len(value) == @(field.type.array_size) \
+             len(value) == @(field.type.array_size))
 @[      end if]@
-            and all([isinstance(v, @(get_python_type(field.type))) for v in value]))
 @[    else]@
-            )
+             all([isinstance(v, @(get_python_type(field.type))) for v in value]))
```

Comment:
Since most of the changes in #129 have been reverted and only contains this single change I would propose to close #129 since it is already included here.